src/main/java/org/jitsi/jigasi/xmpp/CallControlMucActivator.java

```diff
@@ -347,15 +347,34 @@ public void onSessionFailed(AbstractGatewaySession session)
      */
     private void updatePresenceStatusForXmppProviders()
     {
-        SipGateway gateway = ServiceUtils.getService(
+        SipGateway sipGateway= ServiceUtils.getService(
             osgiContext, SipGateway.class);
+        TranscriptionGateway transcriptionGateway = ServiceUtils.getService(
+            osgiContext, TranscriptionGateway.class);
 
         int participants = 0;
-        for(SipGatewaySession ses : gateway.getActiveSessions())
+        int sessions  = 0;
+
+        if(sipGateway != null)
         {
-            participants += ses.getJvbChatRoom().getMembersCount();
+            for(SipGatewaySession ses : sipGateway.getActiveSessions())
```

Comment:
How about collecting all active sessions and then looping over them once (avoid duplicating the counting code)